models/FacilitySMDGLocation.yaml

```
@@ -0,0 +1,45 @@
+type: object
+name: FacilitySMDGLocation
+x-stoplight:
+  id: ejl7tow06gnpb
+description: "\t\nAn interface used to express a location using a `UnLocationCode`."
```

This is not correct - should rather be:
```suggestion
description: "\t\nAn interface used to express a location using a `SMDG Facility Code`."
```